src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Generated/UpdateBeanstalkEnvironment/IsCompleteLambda/IsComplete.py

```diff
@@ -0,0 +1,23 @@
+import boto3
+
+client = boto3.client('elasticbeanstalk')
+
+
+def handler(event, context):
+    request_type = event['RequestType'].lower()
+    props = event['ResourceProperties']
+    applicationName, environmentName, versionLabel = props['ApplicationName'], props['EnvironmentName'], props['VersionLabel']
+
+    if request_type == 'delete':
```

Comment:
Add comment why we don't care about delete

Reply:
Fixed